reference/shaders-hlsl/vert/basic.vert

```diff
@@ -0,0 +1,47 @@
+struct _UBO
+{
+    float4x4 uMVP;
+};
+
+cbuffer UBO
+{
+    _UBO _16;
+};
+uniform float4 gl_HalfPixel;
+
+static float4 gl_Position;
+static float4 aVertex;
+static float3 vNormal;
+static float3 aNormal;
+
+struct InputVert
+{
+    float3 aNormal : TEXCOORD0;
+    float4 aVertex : TEXCOORD1;
+};
+
+struct OutputVert
+{
+    float3 vNormal : TEXCOORD0;
+    float4 gl_Position : POSITION;
+};
+
+void vert_main()
+{
+    gl_Position = mul(transpose(_16.uMVP), aVertex);
```

Comment:
Matrices always have to be transposed to get the maths straight. ANGLE does the very same thing. Would love to find a better and faster solution though but I guess that's very tricky.